dev-resources/tla-demo/docker-compose.yml

```diff
@@ -0,0 +1,240 @@
+# Example Architecture with Two Noisy LRS' and one Transactional LRS
+
+version: "3.9"
```

Comment:
> WARN[0000] /Users/milt/projects/lrsql/dev-resources/tla-demo/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion